packages/components-js/scripts/replaceChunkAndIconLinks.ts

```diff
@@ -0,0 +1,28 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import { getComponentChunkLinks, getIconLinks } from '../dist/components-wrapper/partials';
+import { COMPONENT_CHUNK_NAMES } from '../projects/components-wrapper';
+import { ICON_NAMES } from '@porsche-design-system/assets';
+
+export const chunksLink = getComponentChunkLinks({ components: [...COMPONENT_CHUNK_NAMES] }).replace(
+  /https:\/\/cdn\.ui\.porsche\.com\/porsche-design-system/g,
+  'http://localhost:3001'
+);
+
+export const iconsLink = getIconLinks({ icons: [...ICON_NAMES] }).replace(
+  /https:\/\/cdn\.ui\.porsche\.com\/porsche-design-system/g,
+  'http://localhost:3001'
+);
+
+const replaceChunkAndIconLinks = (): void => {
+  const indexHtmlFilePath = path.resolve(__dirname, '../public/index.html');
```

Comment:
Perhaps we should do this to the file in `www` folder as part of build to not have all those changes in the tracked file.